readthedocs/settings/base.py

```diff
@@ -816,6 +819,10 @@ def DOCKER_LIMITS(self):
         },
     }
 
+    # MailerLite API for newsletter signups
+    MAILERLITE_API_SUBSCRIBERS_URL = 'https://api.mailerlite.com/api/v2/subscribers'
+    MAILERLITE_API_KEY = None
```

Comment:
Ping me and let me know how to get this key. I can open a PR in our -ops repository to set this key.

Reply:
Sent!